dace/frontend/operations.py

```diff
@@ -30,7 +30,16 @@ def timethis(program, title, flop_count, f, *args, **kwargs):
     REPS = int(Config.get('treps'))
     times = [start] * (REPS + 1)
     ret = None
-    for i in range(REPS):
+    print('\nProfiling...')
+    iterator = range(REPS)
+    try:
+        from tqdm import tqdm
+        iterator = tqdm(iterator)
```

Comment:
Looks cool, this might make it a bit nicer :wink: 
```suggestion
        iterator = tqdm(iterator, desc="Profiling")
```

Reply:
Good idea, thanks!